src/components/TodoContainer/TodoContainer.jsx

```diff
@@ -232,9 +226,20 @@ function TodoContainer() {
     setIsAscending(!isAscending);
   };
 
+  const toggleTitleOrder = () => {
```

Comment:
Call the sort function when fetching data, adding a todo, and updating todo to ensure the intended order is preserved (not just when the button is pressed by the user)

Reply:
A good option here is to setup a useEffect to call the sort whenever `todoList` is changed